<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vikass-Laptop.local" failures="1" tests="5" name="Default test" time="74.951" errors="0" timestamp="2024-06-15T13:10:32 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.469">
    <failure type="java.lang.AssertionError" message="TestCase failed Message not matched expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: TestCase failed Message not matched expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at bdd.incubyte.stepDefinitions.RegistrationSteps.check_user_is_registered_successfully(RegistrationSteps.java:53)
at âœ½.Check User is registered successfully.(file:///Users/vikasverma/eclipse-workspace/incubyte/src/test/resources/Features/Login.feature:29)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.477"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.011"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.836"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.943"/>
</testsuite> <!-- Default test -->
